src/App.js

```diff
@@ -1,16 +1,53 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+// import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
+
 
 const App = () => {
+  const [chatData, setChatData] = useState(chatMessages);
+    // {
+    //   'sender':'Vladimir',
+    //   'body':'why are you arguing with me',
+    //   'timeStamp':'2018-05-29T22:49:06+00:00',
+    // }
```

Comment:
Be sure to remove unused code, print tests, and any comments that shouldn't be published before finalizing a project. It's good practice to keep a clean main branch that's production ready. This helps with the readability and maintainability of a project. 🧹😌